runtime/kusama/build.rs

```diff
@@ -19,7 +19,7 @@ use wasm_builder_runner::WasmBuilder;
 fn main() {
 	WasmBuilder::new()
 		.with_current_project()
-		.with_wasm_builder_from_crates("1.0.9")
+		.with_wasm_builder_from_git("https://github.com/paritytech/substrate.git", "8c672e107789ed10973d937ba8cac245404377e2")
```

Comment:
we are using an unreleased feature ?

Reply:
We need this so that the Wasm builds with the correct feature flag. Maybe @bkchr  needs to do another release.